crates/notifier_host/src/proxy/dbus_status_notifier_watcher.rs

```diff
@@ -12,7 +12,11 @@
 //! [Writing a client proxy]: https://dbus2.github.io/zbus/client.html
 //! [D-Bus standard interfaces]: https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces,
 use zbus::proxy;
-#[proxy(interface = "org.kde.StatusNotifierWatcher", assume_defaults = true)]
+#[proxy(
+    default_service = "org.kde.StatusNotifierWatcher",
+    interface = "org.kde.StatusNotifierWatcher",
+    default_path = "/StatusNotifierWatcher"
+)]
```

Comment:
this change from the autogenerated file should probably be documented. i'm currently looking into 71ba5024421dd475e800c55f3efce1dd887cdc56 again, maybe there's more